orchagent/nexthopkey.h

```diff
@@ -45,19 +49,44 @@ struct NextHopKey
             throw std::invalid_argument(err);
         }
     }
+    NextHopKey(const std::string &str, bool overlay_nh)
+	{
+        if (str.find(NHG_DELIMITER) != string::npos)
+        {
+            std::string err = "Error converting " + str + " to NextHop";
+            throw std::invalid_argument(err);
+        }
+        auto keys = tokenize(str, NH_DELIMITER);
+        if (keys.size() != 4)
```

Comment:
In case of overlay "NextHopKey(const std::string &str, bool overlay_nh)" is used, otherwise "NextHopKey(const std::string &str)" is used. Want to keep VxLAN changes separate so that existing code is not impacted.